src/index.js

```diff
@@ -1,12 +1,16 @@
-import QS from 'qs';
-import Cookie from 'js-cookie';
-import EventEmitter from 'events';
-import { debounce, isFunction, assign, omitBy, isNil, uuid } from './utils';
+import QS from "qs";
+import Cookie from "js-cookie";
+import EventEmitter from "events";
+import { debounce, isFunction, assign, omitBy, isNil, uuid } from "./utils";
 
 export const DEBOUNCE_MIN = 500;
 export const DEBOUNCE_MAX = 1500;
 
-const DEFAULT_STRINGIFY_OPTIONS = { arrayFormat: 'brackets', skipNulls: true, encode: true };
+const DEFAULT_STRINGIFY_OPTIONS = {
```

Comment:
not huge deal here, but going forward, let's have cosmetic/styling changes like this in a separate commit to make PR review easier.